controllers/article-controllers.js

```diff
@@ -1,11 +1,32 @@
-
-const {selectArticle} = require("../models/article-model")
+const {
+  selectArticle,
+  selectCommentsByArticleId,
+} = require("../models/article-model");
 
 exports.getArticleByArticleId = (request, response, next) => {
-    const { article_id } = request.params;
-    selectArticle(article_id).then((article) => {
-        response.status(200).send({article})
-    }).catch((err) => {
-        next(err)
+  const { article_id } = request.params;
+  selectArticle(article_id)
+    .then((article) => {
+      response.status(200).send({ article });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
+
+exports.getCommentsByArticleId = (request, response, next) => {
+  const { article_id } = request.params;
+  Promise.all([
+    selectArticle(article_id),
+    selectCommentsByArticleId(article_id),
+  ])
+    .then(([article, comments]) => {
+      if (!article) {
+        return response.status(404).send({ msg: "No article found!" });
+      }
+      response.status(200).send({ article, comments });
```

Comment:
No need to send back the original article unless you're gonna use it in your tests (can't think of a reason why at the moment. Just send the comments!